### ISSUE TITLE
Showing terminal after it being open for long periods of time (over 3 days straight) will prevent it from ever being maximized until terminated.

### ISSUE THREAD

**Issue by AraHaan** [microsoft/terminal#9190]
# Environment

```none
Windows build number: Windows 10 Insider Preview Dev Build: 21301.rs_prerelease.210123-1646 (from the label at the top of the taskbar).
Windows Terminal version (if applicable): I use the latest Terminal Preview Version from the Microsoft Store.

Any other software?
I have Visual Studio 2019, and many other unrelated software installed due to Visual Studio installing it, SQL Express, other Microsoft Store applications, and various versions of the .NET Core SDK / .NET 5 & 6 SDKs.
```

# Steps to reproduce

I have currently no idea what triggers it, it happens at random when having the terminal with 3 Powershell Preview tabs (Powershell Preview from the Microsoft Store that is) which has the directory set and is running an git gui, 1 tab for deleting branches that no longer have a tracking ref on the remote (since my git is set to delete refs if the remote no longer has the branch when pulling), one tab for pulling, commiting, making branches, etc, and last tab to use .NET SDK commands (``dotnet build``, ``dotnet restore``, ``dotnet pack``, etc).

# Expected behavior

The terminal with all the tabs being able to maximize even after being open for those long periods of time.

# Actual behavior

Even when I click on the terminal entry on the taskbar it will refuse the maximize and show the terminal with all 3 tabs.

**Comment by zadjii-msft** [comment]
Wait so I wanna be sure I understand the issue correctly - You currently have the Terminal window minimized, but you can't restore the window by clicking on it in the taskbar?

* Does activating the window with <kbd>alt+tab</kbd> work?
* What about activating the window with the task veiw, <kbd>win+tab</kbd>?

**Comment by AraHaan** [comment]
It seems it does not work with both of them, also I somehow managed to get Discord to do the same thing as well too. 😂

**Comment by zadjii-msft** [comment]
If it's happening to the Discord app as well, then it's probably safe to assume that this isn't an issue unique to the Terminal, yea? 

This is gonna sound weird, but if you alt+tab to the Terminal window, then alt+tab again, is the Terminal window the first in the alt-tab order? 

You could maybe try alt-tabbing to the window, then hit <kbd>alt+space</kbd>, <kbd>m</kbd>, then tap any arrow key, then move the mouse. My theory is that somehow the Terminal window got yeeted into space, and it's restoring to a position that's not on any of the monitors. That "alt+space, m" trick will allow you to start moving the window regardless of where it is on the screen. Maybe that'll summon the window again?

**Comment by AraHaan** [comment]
yep alt tab twice did expose it first, and then it was yeeted into space.

🤔 that alt+space, m was new to me though I did not know I could alt+space, m to force move a window even if I cannot click on the title bar of the window itself due to it being yeeted to space. Something new that I learned that I could have used on yeeted windows that could have saved me from having to terminate them and reopen them and hopefully not be yeeted.
